src/main/drivers/vtx_rtc6705_soft_spi.c

```diff
@@ -136,7 +139,11 @@ void rtc6705SetFrequency(uint16_t channel_freq)
 
 void rtc6705SetRFPower(uint8_t rf_power)
 {
-    rtc6705_write_register(7, (rf_power > 1 ? PA_CONTROL_DEFAULT : (PA_CONTROL_DEFAULT | PD_Q5G_MASK) & (~(PA5G_PW_MASK | PA5G_BS_MASK))));
+	if (!ARMING_FLAG(ARMED) && (vtxSettingsConfig()->lowPowerDisarm == 0xFF) && !failsafeIsActive()) {
```

Comment:
You don't have to check `!ARMING_FLAG(ARMED)` and `!failsafeIsActive()` here. `rf_power` will be `VTX_SETTINGS_ULTRA_LOW_POWER` when `vtxSettingsConfigMutable()->lowPowerDisarm = VTX_SETTINGS_ULTRA_LOW_POWER` and disarmed.